.travis.yml

```diff
@@ -1,162 +0,0 @@
-language: rust
-
-branches:
-  only:
-    - staging
-    - trying
-    - master
```

Comment:
I didn't include 'master' in the ci.yml file as it causes a second CI run on master that appears to be identical to the one on staging that bors verifies with.